src/MetaCompilation.Analyzers/UnitTests/Verifiers/CodeRefactoringVerifier.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.Text;
+using Test.Utilities;
+using Xunit;
+
+namespace MetaCompilation.Analyzers.UnitTests
```

Comment:
Should this file be moved under https://github.com/dotnet/roslyn-analyzers/tree/master/src/Test.Utilities? Namespace should also be changed accordingly.

Reply:
The trouble is that this type derives from `MetaCompilation.Analyzers.UnitTests.DiagnosticVerifier`. Should we move `DiagnosticVerifier` as well?

---
In reply to: [364432464](https://github.com/dotnet/roslyn-analyzers/pull/2588#discussion_r364432464) [](ancestors = 364432464)